src/views/Admin/components/CliftonStrengthsUpload/index.js

```diff
@@ -52,20 +49,34 @@ const CliftonStrengthsUpload = () => {
     CliftonStrengthsService.postCliftonStrengths(formattedData)
       .then((data) => {
         setUploadResults(
-          data.map((r, index) => (
-            <TableRow key={index} sx={{ '&:last-child td, &:last-child th': { border: 0 } }}>
-              <TableCell style={{ fontSize: '1rem' }}>{r.Email}</TableCell>
-              <TableCell style={{ fontSize: '1rem' }}>{r.AccessCode}</TableCell>
+          data.map((uploadResult, index) => (
+            <TableRow key={index}>
+              <TableCell className={styles.strengthstable_cell}>{uploadResult.Email}</TableCell>
+              <TableCell className={styles.strengthstable_cell}>
+                {uploadResult.AccessCode}
+              </TableCell>
               <TableCell
-                style={{
-                  backgroundColor: successResults.some((s) => s === r.UploadResult)
-                    ? gordonColors.secondary.greenShades.secondary
-                    : gordonColors.secondary.red,
-                  color: 'white',
-                  fontSize: '1rem',
-                }}
+                className={
+                  successResults.some((s) => s === uploadResult.UploadResult)
```

Comment:
I'm noticing that we compute this value three times for each uploadResult. Would it make sense to assign to a variable instead and reuse the variable?